spec/board_spec.rb

```diff
@@ -8,4 +8,23 @@
       expect(board.board).to eq([1, 2, 3, 'x', 5, 6, 7, 8, 9])
     end
   end
+
+  describe 'tracking remaining moves' do
```

Comment:
What is the behaviour you want to test. I think it is `making a move` rather than directly `tracking moves`.

Tracking moves is required in order to determine when a game is complete, but I'm not sure there is much value in testing `tracking moves` directly.